tests/mol-tts_single/EPUB/package.opf

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Media Overlays</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2022-02-17</dc:date>
+    <dc:description>Tests whether the Reading System implements Text-to-Speech as a fallback when no audio content is provided. Only one fragment is defined in the SMIL file.</dc:description>
```

Comment:
Nit: we never capitalize Reading System in the descriptions (see https://w3c.github.io/epub-tests/). I don't really care about the content of the test file, but let's try to keep this consistent in test descriptions so the test document is clean.

Reply:
+1